pom.xml

```diff
@@ -106,6 +106,8 @@
     </scm>
 
     <modules>
+        <module>legend-shared-test-reports</module>
```

Comment:
The module order is being changed to address the issue with the nexus staging plugin.
If the last module has a configuration to skip it's deployment, then the deployment of the whole project is skipped:

https://issues.sonatype.org/browse/NEXUS-9138